rholang/src/main/scala/coop/rchain/rholang/interpreter/storage/implicits.scala

```diff
@@ -27,7 +27,7 @@ object implicits {
       }
     }
 
-  def matchListPar[F[_]: Sync: Span]: StorageMatch[F, BindPattern, ListParWithRandom] =
+  implicit def matchListPar[F[_]: Sync: Span]: StorageMatch[F, BindPattern, ListParWithRandom] =
```

Comment:
We should be instantiating `matchListPar` explicitly. Otherwise good luck with finding where it's being wired in, and how many instances there are. So please drop the `implicit`